docs/sources/shared/alerts/alerting_provisioning.md

```diff
@@ -1393,6 +1472,14 @@ Status: Accepted
 
 {{% /responsive-table %}}
 
+### <span id="mute-timing-export"></span> MuteTimingExport
```

Comment:
It's pending to document these properties.  If it is not possible now, I suggest merging this and adding this later in a separate PR.